player/lua/ytdl_hook.lua

```diff
@@ -47,6 +47,7 @@ local tag_list = {
     -- (default --display-tags does not include this name)
     ["description"]     = "ytdl_description",
     -- "title" is handled by force-media-title
+    -- tags don't work with all_formats=yes
```

Comment:
This doesn't actually have anything to do with this commit right? Just something you noticed?

Reply:
Yes I first tried setting it via tags, but then looked into why it wasn't done this way in the first place. They mentioned in the commit message that tags don't work with all_formats, which I confirmed in a test. I thought it would be useful to have some explanation why not everything is done via tags.

Reply:
Could you explain that briefly in the commit message?

Reply:
done